images/step-ca/tests/k8s-test.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+# Function to install step-ca
+install_step_ca() {
+    helm repo add smallstep https://smallstep.github.io/helm-charts/
```

Comment:
nit: In other images, we declare a helm module inside of the test submodule. See `argocd` or `harbor` as a reference